samcli/commands/pipeline/external_links.py

```diff
@@ -0,0 +1,5 @@
+"""
```

Comment:
Just wanted to confirm that we are going to add integ test for validating broken link later.